src/Microsoft.Performance.Toolkit.Engine/EngineCreateInfo.cs

```diff
@@ -14,53 +17,72 @@ namespace Microsoft.Performance.Toolkit.Engine
     /// </summary>
     public sealed class EngineCreateInfo
     {
-        private string extensionDirectory;
-
         /// <summary>
-        ///     Initializes a new instance of the <see cref="EngineCreateInfo"/> 
-        ///     class.
+        ///     Initializes a new instance of the <see cref="EngineCreateInfo"/> class.
         /// </summary>
-        public EngineCreateInfo()
+        /// <param name="extensionDirectory">
+        ///     The extension directory from which the runtime instance is to load plugins.
+        /// </param>
+        /// <remarks>
+        ///     If no extension directory is specified, the current directory is used.
+        /// </remarks>
+        /// <exception cref="InvalidExtensionDirectoryException">
+        ///     Thrown if <paramref name="extensionDirectory"/> is an invalid directory path.
+        /// </exception>
+        public EngineCreateInfo(string extensionDirectory = null)
```

Comment:
thoughts on a separate parameterless ctor instead of using a default?

Reply:
sure, i'm ok with that.